include/rocksdb/wal_filter.h

```diff
@@ -30,6 +32,20 @@ class WalFilter {
 
   virtual ~WalFilter() {}
 
+  // Provide ColumnFamily->LogNumber map to filter
+  // so that filter can determine whether given record
+  // is applicable to given column family.
+  // We also pass in name->id map as this is known at
+  // recovery time and write batch callbacks happen
+  // in terms of column family id.
+  //
+  // @params cf_lognumber_map column_family_id to lognumber map
```

Comment:
Need more description here. What this mapping mean. I guess it means the earliest log number where this column family has data not flushed to SST files?